src/components/Dropdown/PrioritySelector/index.ts

```diff
@@ -0,0 +1,6 @@
+export {
+    PrioritySelector,
+    Props as PrioritySelectorProps,
+    PriorityItem,
+    Priority,
```

Comment:
Should we say Type somewhere? It might be only me, but I'm always confused if things are JS or TS.